core/divecomputer.cpp

```diff
@@ -38,13 +38,13 @@ bool DiveComputerNode::changesValues(const DiveComputerNode &b) const
 
 const DiveComputerNode *DiveComputerList::getExact(const QString &m, uint32_t d)
 {
-	auto it = std::lower_bound(dcs.begin(), dcs.end(), DiveComputerNode { m, d , {}, {}, {} } );
+	auto it = std::lower_bound(dcs.begin(), dcs.end(), DiveComputerNode{m, d, {}, {}, {}});
```

Comment:
I think @neolit123 (sorry if I mis-attribute) complained that I should add spaces inside braces.

In consistency with the changes of `core/settings/qPrefDisplay.h` I would propose `DiveComputerNode{ m, d, {}, {}, {} }`, i.e. no space after the class-name, but spaces after `{` and before `}`

Reply:
> I think @neolit123 (sorry if I mis-attribute) complained that I should add spaces inside braces.
i don't think i reviewed this PR:
https://github.com/Subsurface-divelog/subsurface/pull/1386
maybe somewhere else?

the way we initialize in C is:
`int ar[] = { 0, 1, 2 };`
so `DiveComputerNode{ m, d, {}, {}, {} }` seems better.

Reply:
Yes, I think it was in the context of a lambda, which is a bit of a different situation.

Reply:
Some of this is of course black magic hidden somewhere in the logic of clang-format.
What I would say is that we tend to have parenthesis without spaces `bla(3, "no")`, but braces with no code behind them (so the opening brace that starts a code block is typically followed by a newline) - but in the rare instances where we went for the "tight" code model `void bla(int x) { return x + 1; }` we typically have spaces there as well.
I like spaces better and this was one of the changes I wasn't as excited about. But that means someone needs to figure out how to teach `whitespace.pl` how to do this right.

Reply:
should we merge this as is?

Reply:
I think so. And then fix `whitespace.pl` and go over things again to make them consistent.

But then I may be biased.